src/sidebar/components/test/annotation-header-test.js

```diff
@@ -93,5 +93,31 @@ describe('sidebar.components.annotation-header', function () {
         assert.deepEqual(ctrl.displayName(), 'Bill Jones');
       });
     });
+
+    describe('#thirdPartyUsernameLink', () => {
+      it('returns the thirdPartyUsernameLink if set', () => {
```

Comment:
There isn't a property called "thirdPartyUsernameLink" in the settings object. Perhaps "returns the custom username link if set"?